src/docs/perf/app-size.md

```diff
@@ -88,6 +88,12 @@ The resulting IPA file for the `example/helloworld` app
 -rw-r--r--  1 userName  primarygroup   8.3M Oct 25 13:47 build/app.ipa
 ```
 
+A closer result can be obtained by creating a release archive as described in
+the [iOS create build archive instructions][]. If bitcode is enabled on your
+project, you will also have the option to rebuild from bitcode. You can also
```

Comment:
Describe what 'rebuild from bitcode' means and whether the user should or should not do so.

Reply:
Added a line.